frontend/app/model/Eventbrite.scala

```diff
@@ -187,6 +189,10 @@ object Eventbrite {
       case t: InternalTicketing => t
     }
 
+    val limitedAvailabilityText = "Last tickets remaining"
```

Comment:
Potentially this text could sit in the fragment even though it is repeated. Would then be more explicit?